docs/outlook/autolaunch-store-options.md

```diff
@@ -0,0 +1,73 @@
+
+## Deploy add-in with flight code
+
+After the flight codes are set, you'll receive the URL from the app certification team. You can then share the URL with admins privately.
+
+To deploy the add-in, the admin can use the following steps.
+
+- Sign in to admin.microsoft.com or AppSource.com with your Microsoft 365 admin account. If the add-in has Single sign-on (SSO) enabled, global admin credentials are needed.
+- Open the flight code URL into a web browser.
+- On the add-in listing page, select **Get it now**. You should be redirected to the integrated app portal.
+
+## Unrestricted AppSource listing
+
+If your add-in doesn't use event-based activation for critical scenarios (that is, your add-in works well without automatic invocation), consider listing your add-in in AppSource without any special flight codes. If an end-user gets your add-in from AppSource, automatic activation won't happen for the user. However, they can use other components of your add-in such as a task pane or UI-less command.
+
+> [!IMPORTANT]
+> This is a temporary restriction. In future, we plan to enable event-based add-in activation for end-users who directly acquire your add-in.
+
+## Update existing add-ins to include event-based activation
+
+You can update your existing add-in to include event-based activation then resubmit it for validation and decide if you want a restricted or unrestricted AppSource listing.
+
+After the updated add-in has been approved, organization admins who have already deployed the add-in will receive an update message in the admin portal. The message advises the admin about the event-based activation changes. After the admin accepts the changes, the update will be deployed to end-users.
+
+For end-users who installed the add-in on their own, the event-based activation feature won't work even after the add-in has been updated.
+
+## Admin consent for installing event-based add-ins
+
+Whenever an event-based add-in is deployed from the **Integrated Apps** section of the admin center, the admin gets details about the add-in's event-based activation capabilities in the deployment wizard. The details appear in the **App Capability** section. The admin should see all the events where the add-in can automatically activate.
+
+![Screenshot of "Accept permissions requests" screen when deploying a new app.](../images/outlook-deploy-accept-permissions-requests.png)
+
+Similarly, when an existing add-in is updated to event-based functionality, the admin sees an "Update Pending" status on the add-in. The updated add-in is deployed only if the admin consents to the changes noted in the **App Capability** section, including the set of events where the add-in can automatically activate.
+
+Each time you add any new `LaunchEvent Type` to your add-in, admins will see the update flow in the admin portal and need to provide consent for additional events.
+
```

Comment:
The screenshot with update flow should be added here: 

![MicrosoftTeams-image (22)](https://user-images.githubusercontent.com/49477715/125277798-89cfd800-e32f-11eb-9912-82e254d71efb.png)